sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/DynamicDestinationsHelpers.java

```diff
@@ -28,13 +34,20 @@
 import org.apache.beam.sdk.options.ValueProvider;
 import org.apache.beam.sdk.options.ValueProvider.NestedValueProvider;
 import org.apache.beam.sdk.transforms.SerializableFunction;
+import org.apache.beam.sdk.util.BackOffAdapter;
```

Comment:
This broke the build:
```
08:15:19 > Task :sdks:java:io:google-cloud-platform:compileJava
08:15:19 /home/jenkins/jenkins-slave/workspace/beam_PostCommit_Java_Nexmark_Dataflow/src/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/DynamicDestinationsHelpers.java:40: error: cannot find symbol
08:15:19 import org.apache.beam.sdk.util.BackOffAdapter;
```

Reply:
@apilloud could you file an issue or otherwise broadcast this?

Reply:
It is fixed now by #8726. Please rebase.

Reply:
Great, thanks for the fix :+1: